Formula/circleci.rb

```diff
@@ -0,0 +1,47 @@
+class Circleci < Formula
+  desc "Tool which enables to reproduce CircleCI build locally"
+  homepage "https://circleci.com/docs/2.0/local-jobs/"
+  url "https://circle-downloads.s3.amazonaws.com/releases/build_agent_wrapper/circleci"
+  version "0.0.4579-9c6473a"
+  sha256 "2db865b97d490914e7fb6a056f94d80b1cc9e3b2ed9c09723b529d673c131d7a"
+
+  bottle :unneeded
+
+  depends_on "docker"
+
+  def install
+    bin.install "circleci"
```

Comment:
Needs to be built from source if this is using `go`, which I suspect it very well might be.

Reply:
> You can update your local build tool with circleci update.

Also suspect Homebrew may want a way to disable that, looking at things like `heroku`.

Reply:
The program is a shell script that invokes a Docker container, so how can I build it from source ?

Reply:
The update process downloads a newer Docker image but does not update the CLI itself

Reply:
> The program is a shell script that invokes a Docker container, so how can I build it from source ?

Oh interesting, I assumed it was a `go` binary or something, as has become incredibly popular. If it's simply a shell script it's obviously fine to not build from source.

> The update process downloads a newer Docker image for the build agent but does not update the CLI itself.

That should _probably_ be okay then, I assume.